datalad/customremotes/ria_utils.py

```diff
@@ -0,0 +1,158 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Helper for RIA stores
+
+"""
+
+
+class UnknownLayoutVersion(Exception):
+    pass
+
+
+known_versions_objt = ['1', '2']
```

Comment:
what is `_objt`.  having comments describing those constants (are they? if so - make `tuple` ?) would help new to the code developers to orient themselves

Reply:
Added comments to the constants, but made the `tuple` suggestion a TODO only. This would require a bit more RF'ing that shouldn't delay merge right now, I think. But: Certainly useful approach. However, may be better elsewhere (define versions as `(label, description)` and reference here by `label` only) - not sure yet.